sdk/tables/azure-tables/src/serialization.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { EdmGuid } from "./EdmGuid";
+import { EdmInt64 } from "./EdmInt64";
+import { isUint8Array, encodeByteArray, decodeString } from "./utils/bufferSerializer";
+
+const Edm = {
+  DateTime: "Edm.DateTime",
+  Int64: "Edm.Int64",
+  Guid: "Edm.Guid",
+  Binary: "Edm.Binary"
+};
+
+export function serialize(obj: any): object {
```

Comment:
This function doesn't really take `any` though, right? If I called it with a number, I think it would fail. It seems to take `object`?